src/github.com/stellar/go-horizon/resource_account.go

```diff
@@ -62,6 +76,33 @@ func NewAccountResource(ac db.AccountRecord) AccountResource {
 	// add native balance
 	balances[len(ac.Trustlines)] = BalanceResource{Type: "native", Balance: AmountToString(ac.Balance)}
 
+	// signers
+	signers := make([]SignerResource, len(ac.Signers))
+
+	for i, s := range ac.Signers {
+		signers[i] = SignerResource{Address: s.Publickey, Weight: s.Weight}
+	}
+
+	// thresholds
+	reader := strings.NewReader(ac.Thresholds)
+	b64r := base64.NewDecoder(base64.StdEncoding, reader)
+	var xdrThresholds xdr.Thresholds
+	_, err := xdr.Unmarshal(b64r, &xdrThresholds)
+
+	var thresholdsMap map[string]byte
+	if err == nil {
+		thresholdsMap = make(map[string]byte)
+		thresholdsMap["threshold_master_weight"] = xdrThresholds[0]
+		thresholdsMap["threshold_low"] = xdrThresholds[1]
+		thresholdsMap["threshold_med"] = xdrThresholds[2]
+		thresholdsMap["threshold_high"] = xdrThresholds[3]
+	}
+
+	// flags
+	flagsMap := make(map[string]bool)
+	flagsMap["auth_required_flag"] = IsTrue(ac.Flags & 1)
+	flagsMap["auth_revocable_flag"] = IsTrue(ac.Flags & 2)
```

Comment:
A couple of notes on this section:

1.  Magic Numbers:  If I was writing this code, I would include constants in the `record_core_account.go` file that specify the masks for the various flags.  You can see an example from here: http://stackoverflow.com/questions/14426366/what-is-an-idiomatic-way-of-representing-enums-in-golang

```go
const (
   FlagAuthRequired = 1 << iota
   FlagAuthRevoked = 1 << iota
)
```

2.  I would add methods to `*CoreAccountRecord`: `IsAuthRequired() bool` and `IsAuthRevocable() bool`

Adding these methods puts the logic of "what flags are set" into the model layer of the application rather than the front end.  This allows for more re-use down the line.